forms-web-app/tests/unit/routes/appellant-submission/index.test.js

```diff
@@ -20,15 +13,6 @@ describe('routes/appellant-submission/index', () => {
   });
 
   it('should define the expected routes', () => {
-    expect(use).toHaveBeenCalledWith(appealStatementRouter);
```

Comment:
these test were nonsense

each of these routers were anonymous mocks. Could have done 

`expect(use).toHaveBeenCalledWith(jest.fn());` x 11 and still got a pass.

Reply:
just read that and snorted coffee on my keyboard. thanks.